kv/mpp.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kv
+
+import (
+	"context"
+
+	"github.com/pingcap/kvproto/pkg/mpp"
+)
+
+// MPPTaskMeta means the meta info such as location of a mpp task.
+type MPPTaskMeta interface {
+	// GetAddress indicates which node this task should execute on.
+	GetAddress() string
+}
+
+// MPPTask means the minimum execution unit of a mpp computation job.
+type MPPTask struct {
+	Meta    MPPTaskMeta // on which store this task will execute
```

Comment:
meta -> location?

Reply:
localtion is fine, but I think meta has a broader meaning. Currently this object only serves as an address provider, but we can leave more space to provide other information, like concurrency number.

Reply:
so the comment is not exact: where to run, how many threads and other runtime info.

Reply:
Yes, but currently it's sole function is to decide the store/location. It's accurate at least for now.